db/migrate/20231213110952_create_gias_schools.rb

```diff
@@ -0,0 +1,12 @@
+class CreateGiasSchools < ActiveRecord::Migration[7.1]
+  def change
+    create_table :gias_schools, id: :uuid do |t|
```

Comment:
Looking at @simonwhatley's prototype is it worth storing more information about the school? 
https://itt-mentoring-beta-94341ca35a2c.herokuapp.com/support/organisations/049fc44e-6876-4884-8a77-d363173ce8d2/details 
Publish store a lot more information. https://github.com/DFE-Digital/publish-teacher-training/tree/main/db

Reply:
That's a good shout, I'll include the columns in that prototype

Reply:
There is no email address in the Gias CSV file, I've added the rest though.